app/javascript/shared/components/useKeyboardShortcuts.jsx

```diff
@@ -1,4 +1,4 @@
-import { useEffect } from "preact/hooks";
+import { useState, useEffect, useCallback } from "preact/hooks";
 import PropTypes from 'prop-types';
 import { h } from 'preact';
```

Comment:
refactor (non-blocking): There is no JSX in this file so it can be renamed to `useKeyboardShortcuts.js`.

refactor (non-blocking): I think this is remnants of a previous PR when there was JSX in this file. It'd be good to get it cleaned up since these hooks have been reworked.
 
```suggestion
```